src/inc/msquic.h

```diff
@@ -1605,6 +1605,35 @@ QUIC_STATUS
     _In_opt_ void* ClientSendContext
     );
 
+//
+// Connection Pool API
+//
+
+//
+// Creates a simple connection pool with NumberOfConnections connections
+// all with the same Context and Handler, and puts them in the
+// caller-supplied array.
```

Comment:
Mention that the goal of this pool is to evenly spread these connections across RSS as much as possible.